scss/mixins/_utilities.scss

```diff
@@ -0,0 +1,184 @@
+// Utility Mixins 
+// 
+// Weight: -90
+//
+// Styleguide mixins
+
+// u-visually-hidden() 
+//
+// Used to visually hide components but keep them visible to screenreaders. Also includes a utility class .u-visually-hidden.
+//
+// Styleguide mixins.visually-hidden
+
+@mixin u-visually_hidden {
+  position: absolute !important;
+  top: auto;
+  left: -9999px !important;
+}
+
+.u-visually-hidden {
+  @include u-visually-hidden();
+}
+
+.u-float-left {
+  float: left;
+}
+
+.u-float-right {
+  float: right;
+}
+
+// u-font-color()
+//
+// Used to set the color of all child elements
+//
+// $color     - The color of the font and (if applicable) border-color
+//
+// Styleguide mixins.font-color
+
+@mixin u-font-color($color) {
+  color: $color;
+
+  h1, 
+  h2, 
+  h3, 
+  h4, 
+  h5, 
+  h6,
+  a, 
+  p, 
+  span, 
+  li, 
+  dt, 
+  dd, 
+  label,
+  blockquote,
+  .label {
+    color: $color;
+  }
+
+  h1,
+  h6,
+  a {
+    border-color: $color;
+  }
+
+  .t-ruled {
+    border-color: $color;
+  }
+}
+
+// u-form-color()
+//
+// Used to set the color of the form elements
+//
+// $background-color      - The color of the background of the inputs
+// $contrast-color        - Border and font colors
+// $focus-color           - Focus color
+// $icon-color            - A string that will be appended to the background-image url for checkboxes (e.g. 'primary', 'secondary')
+//
+// Styleguide mixins.font-colo
+
+@mixin u-form-color($background-color, $contrast-color, $focus-color, $icon-color) {
+  #{$all-text-inputs},
+  select,
+  select[multiple=multiple],
+  textarea {
+    color: $contrast-color;
+    border-color: $contrast-color;
+    background-color: $background-color;
+
+    &:focus {
+      border-color: $focus-color;
+      color: $focus-color;
+    }
+  }
+  
+  select {
+    background-image: url('../img/i-down--' + $icon-color + '-contrast.png');
+    background-image: url('../img/i-down--' + $icon-color + '-contrast.svg');
+  }
+
+  input[type="checkbox"] + label::before,
```

Comment:
It might be good to remove the `input` parts of these because this is in a mixin. So just `[type="checkbox"]`.